crates/red_knot_python_semantic/src/types/diagnostic.rs

```diff
@@ -320,6 +320,18 @@ declare_lint! {
     }
 }
 
+declare_lint! {
+    /// ## What it does
+    /// Checks for invalid parameters to `typing.Annotated`.
+    ///
+    /// TODO #14889
```

Comment:
It would be nice to avoid adding more TODO comments to this file. Can you add a short description of this check, similar to the one here:

https://github.com/astral-sh/ruff/blob/c3a64b44b7ab8742f4fe3630cd9751104d9f78d2/crates/red_knot_python_semantic/src/types/diagnostic.rs#L60-L77